comments.php

```diff
@@ -0,0 +1,56 @@
+<?php
+/**
+ * Comments template.
+ *
+ *  @package WPAMPTheme
+ */
+
+/*
+ * If the current post is protected by a password and
+ * the visitor has not yet entered the password we will
+ * return early without loading the comments.
+ */
+if ( post_password_required() ) {
+	return;
+}
+
```

Comment:
This is outside the diff, but [could you please add](https://github.com/xwp/travel/blob/111aa870a7c1f09765a683582de1eefdfe4e08b6/includes/class-amp-travel-cpt.php#L158) `'comments'` to the CPT `'supports'` argument?